taxonomies/product/categories.txt

```diff
@@ -5,18 +5,142 @@
 # - carbon_impact_fr_impactco2:en: co2 equivalent in kg per unit (1 product) from https://impactco2.fr/
 # - carbon_impact_fr_impactco2_link:en: URL for the category on https://impactco2.fr/
 
+
+
 en: Furniture
 
+< en:Furniture
+en: Wooden chair
+fr: Chaise en bois
+carbon_impact_fr_impactco2:en: 18.6
+carbon_impact_fr_impactco2_link:en: https://impactco2.fr/outils/mobilier/chaiseenbois
+unit_name:en: Wooden chair
```

Comment:
if unit_name is specific to Impact CO2, then we should prefix the property name.

Reply:
I have no idea what it's for.

Reply:
probably display ?

Reply:
yes it's for display, it's useful in particular for products that have sub products....  but as it comes from carbon impact, it would be better to prefix the properties

Reply:
also in other flavors, we use plural for category names

Reply:
so `impactco2: unit_name:en: Wooden chair` ? @stephanegigandet

Reply:
https://github.com/openfoodfacts/openfoodfacts-server/blob/0c05077e3a7c007661b31b61f208c7bde55014e5/lib/ProductOpener/KnowledgePanels.pm#L962